fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -271,22 +309,25 @@ private PollingLocationResponse process(BulkExportJobExecutionResponse response)
         // TODO: Convert to yyyy-MM-dd'T'HH:mm:ss
         result.setTransactionTime(response.getLastUpdatedTime());
         // Compose outputs for all exported ndjson files from the batch job exit status,
-        // e.g, Patient[1000,1000,200]:Observation[1000,1000,200]
+        // e.g, Patient[1000,1000,200]:Observation[1000,1000,200],
+        //      COMPLETED means no file exported.
         String exitStatus = response.getExitStatus();
-        List<String> ResourceTypeInfs = Arrays.asList(exitStatus.split("\\s*:\\s*"));
-
-        List< PollingLocationResponse.Output> outPutList = new ArrayList< PollingLocationResponse.Output>();
-        for (String resourceTypeInf: ResourceTypeInfs) {
-            String resourceType = resourceTypeInf.substring(0, resourceTypeInf.indexOf("["));
-            String resourceCounts[] = resourceTypeInf.substring(resourceTypeInf.indexOf("[")+1, resourceTypeInf.indexOf("]"))
-                    .split("\\s*,\\s*");
-            for (int i = 0; i < resourceCounts.length; i++) {
-                String downloadUrl =
-                        baseCosUrl + "/" + bucket + "/job" + jobId + "/" + resourceType + "_" + (i+1) + ".ndjson";
-                outPutList.add(new PollingLocationResponse.Output(resourceType, downloadUrl, resourceCounts[i]));
+        if (!exitStatus.contentEquals("COMPLETED")) {
+            List<String> ResourceTypeInfs = Arrays.asList(exitStatus.split("\\s*:\\s*"));
+
+            List< PollingLocationResponse.Output> outPutList = new ArrayList< PollingLocationResponse.Output>();
```

Comment:
FYI - you can use the `<>` diamond operator here.  It infers the types.  we're split in the various patterns we use across the code base.

Reply:
wanted to be more accurate, but make sense to simply it.